### DISCUSSION TITLE
add ser2net client connection to server

### DISCUSSION THREAD

**Discussion by kissandr** [arendst/Tasmota#14756] [Ideas]
I'm looking for a way where tasmota lies behind a NAT, and connects it's serial port via TCP to a public server's IP and port number.
Current tcp bridge module only works as a server, waiting for incoming connections.

**Comment by s-hadinger** [comment]
This feature is not supported right now, unless someone adds it and submits a PR.

**Reply by kissandr** [reply]
Follow up PR #14874 
It's already in the development branch! :1st_place_medal:
